main.py

```diff
@@ -1,10 +1,50 @@
-import init_django_orm  # noqa: F401
-
+import json
+import os
+import django
+from django.utils import timezone
 from db.models import Race, Skill, Player, Guild
 
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "myproject.settings")
+django.setup()
+
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for player_name, player_data in players_data.items():
+        race_data = player_data.get("race", {})
+        race_name = race_data.get("name")
+        race_description = race_data.get("description", "")
+
+        race, _ = (Race.objects.get_or_create
+                   (name=race_name, defaults={"description"
+                                              : race_description}))
```

Comment:
```suggestion
        race, _ = Race.objects.get_or_create(
            name=race_name, 
            defaults={"description": race_description}
        )
```
please fix in other places